odl/tomo/operators/xray_trafo.py

```diff
@@ -39,19 +41,22 @@
 
 _SUPPORTED_BACKENDS = ('astra', 'astra_cpu', 'astra_cuda')
 
-__all__ = ('DiscreteXrayTransform', 'DiscreteXrayTransformAdjoint')
+__all__ = ('XrayTransform', 'XrayTransformAdjoint',)
```

Comment:
Currently it can only be created as `XrayTransform.adjoint` I say we keep this until that is fixed?